### PR TITLE
tools: add a .npmignore file

### PR DESCRIPTION
**Pull request by gibfahn**
- Ignore what's in .gitignore + test/

Fixes: https://github.com/nodejs/citgm/issues/366

##### Checklist

- [x] contribution guidelines followed [here](https://github.com/nodejs/citgm/blob/master/CONTRIBUTING.md)
- [x] commit message follows commit guidelines

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+test/
+coverage/
+.nyc_output/
```
Comment (gdams) [inline .npmignore]: Can you include the .github folder

**Comment by gdams** [comment]
@MylesBorins how do you feel about not including the tests ?

File: .npmignore (hunk 1 above)
Comment (gibfahn) [inline .npmignore]: Done

**Comment by gibfahn** [comment]
Stats for file size in https://github.com/nodejs/citgm/issues/366

**Comment by MylesBorins** [comment]
I think we should not ignore test. Without it citgm does not pass citgm 🤣

**Comment by gibfahn** [comment]
@MylesBorins Shouldn't citgm be testing itself from GitHub?

**Comment by MylesBorins** [comment]
¯\\\_(ツ)\_/¯

I'm just not convinced these types of micro optimizations are necessary. React alone is 45 MB, so shaving 200kb off the npm install isn't going to affect the bottom line of our test suite

**Comment by gibfahn** [comment]
I just thought that not including `test/` in your "production" npm module was the normal thing to do.

I added the `.npmignore` because @gdams raised an issue.

Happy to change this to whatever you want, but I don't see the need to include this stuff in the npm module.

**Comment by gdams** [comment]
screen shot 2017-02-27 at 15 41 18

**Comment by MylesBorins** [comment]
landed in b8a35ee073.

I may bike shed about tests another time 😄

**Pull request closed without merging**